### DISCUSSION TITLE
Can this library deploye on Nvidia Jetson nano？

### DISCUSSION THREAD

**Discussion by shiyi23** [NVIDIA/cccl#1041] [Thrust]
I want to use the library on jetson nano, it is aarch64 architect.

**Comment by alliepiper** [comment]
It should. Let us know if you have any issues.

**Reply by shiyi23** [reply]
> It should. Let us know if you have any issues.

Thanks you!

**Comment by carbonphyber** [comment]
Yes, it does. The standard OS image contains `nvcc` and the Thrust library.
